src/models/oryp7/README.md

```diff
@@ -0,0 +1,58 @@
+# Oryx Pro (oryp7)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Oryx Pro](./img/oryp7.webp)
+
+The System76 Oryx Pro is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel Comet Lake CPUs
+        - [Intel i7-10870H](https://ark.intel.com/content/www/us/en/ark/products/208018/intel-core-i7-10870h-processor-16m-cache-up-to-5-00-ghz.html)
+- Chipset
+    - [Intel HM470](https://ark.intel.com/content/www/us/en/ark/products/203697/intel-hm470-chipset.html)
+- BIOS
+    - MX25L12873F flash chip running [System76 Open Firmware](https://github.com/system76/firmware-open)
```

Comment:
Is this correct? U66 is shown as GD25R127D in the schematics. But MX25L12873F is listed below the GigaDevice chip and `chip.txt` has MX25L12805D...Does the chip package say Macronix?

Reply:
@crawfxrd I was trying to match what you provided for oryp6 (`MX25L12872F`) in https://github.com/system76/tech-docs/pull/62. The oryp6 Clevo manual shows:

![oryp6-bios](https://user-images.githubusercontent.com/7199422/109999232-3e495580-7ccf-11eb-8016-9fccc32b9248.png)

The oryp7 Clevo manual shows:

![oryp7-bios](https://user-images.githubusercontent.com/7199422/109999246-40abaf80-7ccf-11eb-8cd5-a67bfdf6d907.png)

I also found a [datasheet](https://media.digikey.com/pdf/Data%20Sheets/Macronix/MX25L12873F.pdf) for `MX25L12873F` similar to the one you linked for the `MX25L12872F`. I do see what you're talking about with the schematic being different.

> Does the chip package say Macronix?

No, but it does say `MX25L12873F`. Here is a picture:

![P1030647](https://user-images.githubusercontent.com/7199422/110000544-a77d9880-7cd0-11eb-8983-3e2ac3356a28.JPG)

(This is labeled U66, one of the 6's is cut off but it's definitely the BIOS flash chip as I was able to flash it with a CH341A previously.)